src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -53,6 +53,12 @@ namespace dpdk {
     }
 
     bool done = false;
+    const char *hexstring= c->_conf->get_val<std::string>("ms_dpdk_coremask").c_str();
+    int num = (int)strtol(hexstring, NULL, 0);
+    int coremaskbit =bitCout(num);
```

Comment:
this does not compile. 

could you s/bitCount/bitcount/ ,and add a space after `=`?

and might want to use `ceph_assert()`, as `assert()` could be optimized out in future.